addons/web/static/src/core/py_js/py_date.js

```diff
@@ -544,15 +548,62 @@ export class PyRelativeDelta {
     static create(...args) {
         const delta = new PyRelativeDelta();
         const namedArgs = parseArgs(args, argsSpec);
```

Comment:
I think argsSpec is useless here

Reply:
It used for unnamedArgs and I don't think that the API of relativedelta is consistent with that